### DISCUSSION TITLE
Tutorial like "Getting Started"

### DISCUSSION THREAD

**Discussion by goto40** [RemedyIT/ridl#131] [Q&A]
Thank you very much for sharing this nice-looking project! IT seems to cover my needs in terms of features (make IDL models available as AST)!

Do you think it is possible to create tutorial like "Getting started" walk-through: What needs to be done, to, e.g., create a command line application to generate a list of structures and attributes? A minimal example would probably do the thing.

This would ease new developers to get involved and to get comfortable with your design decisions.

Alternatively: give hint where to focus in your existing open source implementations. However, these seem to be rather large in terms of the code base and may be cluttered with distracting details not relevant when learning the `ridl` framework.

**Comment by jwillemsen** [comment]
Currently we don't have plans to create a new tutorial for RIDL. Development is driven by customer needs and internal research. I would recommend you to have a look at the existing backends, including the small test backend which is part of this repository. The test backend is very minimal, probably doesn't work on all IDL files, but maybe a starting point? 

See our website at https://www.remedy.nl/services/opensource-support for an overview of our open source support services.

**Reply by goto40** [reply]
Thank you for the hints.
